src/ConfigGen/Program.cs

```diff
@@ -230,6 +231,9 @@ public PropertySpecification(PropertySpecification other)
         public string Description { get; set; }
         public string Type { get; set; }
         public string AliasFor { get; set; }
+
+        public int CompareTo(object obj)
+            => Name.CompareTo(((PropertySpecification)obj).Name);

```

Comment:
Should this support aliases? I.e., `PropertySpecification("acks") == PropertySpecification("request.required.acks")`?

Reply:
no, they're de-duped and that's done elsewhere. the semantics we want for this comparison is this.